app/src/main/java/cm/aptoide/pt/analytics/analytics/AnalyticsManager.java

```diff
@@ -46,53 +60,139 @@ public void logEvent(Map<String, Object> data, String eventName, Action action,
     }
   }
 
+  /**
+   * <p> Log a {@code url} to the {@link HttpKnockEventLogger}.</p>
+   *
+   * @param url The url to log.
+   */
   public void logEvent(@NonNull String url) {
     knockEventLogger.log(url);
   }
 
+  /**
+   * <p>Setup the {@code EventLogger(s)} that need to be setup, like AptoideBiEventLogger</p>
+   */
```

Comment:
Phrasing. Would go with something like "Setup the required EventLoggers, like AptoideBiEventLogger"